administrator/language/de-DE/de-DE.com_banners.ini

```
@@ -85,7 +85,7 @@ COM_BANNERS_FIELD_CLICKURL_DESC="Die Ziel-URL für den Klick auf das Banner."
 COM_BANNERS_FIELD_CLICKURL_LABEL="Zieladresse"
 COM_BANNERS_FIELD_CLIENT_DESC="Einen Kunden für dieses Banner auswählen."
 COM_BANNERS_FIELD_CLIENT_LABEL="Kunde"
-COM_BANNERS_FIELD_CLIENT_METAKEYWORDPREFIX_DESC="Bei passenden Meta-Schlüsselwörtern, nur nach Meta-Schlüsselwörtern mit diesem Präfix (verbessert die Leistung) suchen."
+COM_BANNERS_FIELD_CLIENT_METAKEYWORDPREFIX_DESC="Bei passenden Schlüsselwörtern, nur nach Schlüsselwörtern mit diesem Präfix (verbessert die Leistung) suchen."
 COM_BANNERS_FIELD_CLIENT_METAKEYWORDPREFIX_LABEL="Schlüsselwörter-Präfix"
 COM_BANNERS_FIELD_CLIENT_METAKEYWORDS_DESC="Hier die Meta-Schlüsselwörter für den Bannerkunden eingeben."
```

Hier ist noch mehr `Meta-Schlüsselwörter`

```suggestion
COM_BANNERS_FIELD_CLIENT_METAKEYWORDS_DESC="Hier die Schlüsselwörter für den Bannerkunden eingeben."
```